tests/phpstan/phpstan-bootstrap.php

```diff
@@ -0,0 +1,6 @@
+<?php declare(strict_types=1);
+/**
+ *To help phpstan dealing with LogicException in Common\User\User.php
+ */
+
+define('GRAV_USER_INSTANCE', 'FLEX');
```

Comment:
Nope, do not do this. The logic exception is there by purpose; Grav needs to have configuration loaded before the compatibility class can be loaded.

Reply:
it's about static analysis,  not a formal bootstrap for running the app. not perfect but at some point the analysis must know which class to use. 

could be solved with a proper interface, but it's good enough. Let me know if you have a better idea.

Reply:
Oh, missed that it was under tests. I guess you can use that then, I'm going to mark this isue solved.